src/index.html

```diff
@@ -1,9 +1,39 @@
 <html>
   <head>
     <title>VIPER</title>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
1. This layout contains all necessary elements, but it doesn't scale well. For now it is OK, but I'd expect the interface to look and scale like original VIPER as a result of this milestone.
2. The logo is nice, but I think it is not necessary - we will not waste precious space to display an image. Hope you agree :)